getnacelle/nuxt-reference-store
# Ld 1111 content

**What This PR Does**
- Adds `/pages/_slug.vue` with hail frequency data
- Adds dynamic section component to render the pages sections

**How to Test**
- spin up project and visit `/pages/about`
- spin up sanity on branch `LL-1183_studio-scaffold`
- check that the pages section `HeroBanner` is using cms data

**Notes**
- there are some section components that are not being used in this PR.  They will be part of page specific PRs.  You can ignore them here.
- There is a dummy data file `data/content.js` which you can ignore.  It will be removed in future PR
